.mega-linter.yml

```diff
@@ -0,0 +1,30 @@
+---
+# don't use Mega-Linter to test the reports Mega-Linter created
+ADDITIONAL_EXCLUDED_DIRECTORIES: [ report, megalinter-reports ]
```

Comment:
Ironically, perhaps, MegaLinter doesn't like this file: there's a few `12:11 [brackets] too many spaces inside brackets` warnings here. Can we fix real quick?

Reply:
You know what's also funny?  Megalinter, when `APPLY_FIXES` is set, will automatically get stuff like that.

But yes, absolutely, I'll fix it right now.